src/components/StoryFooter/Buttons.js

```diff
@@ -127,10 +127,15 @@ export default class Buttons extends React.Component {
     ));
     const upVotesDiff = upVotes.length - upVotesPreview.length;
     const upVotesMore =
-      upVotesDiff > 0 &&
-      intl.formatMessage(
-        { id: 'and_more_amount', defaultMessage: 'and {amount} more' },
-        { amount: upVotesDiff },
+      upVotesDiff > 0 && (
+        <p>
+          <a role="presentation" onClick={this.handleShowReactions}>
+            {intl.formatMessage(
```

Comment:
Correct me if I'm wrong, but I think you can use <FormattedMessage /> component for this

Reply:
thanks for that @jm90m . I'll try it. I'm still not familiar with the components available.